contracts/SnappAuction.sol

```diff
@@ -56,9 +75,61 @@ contract SnappAuction is SnappBase {
         return auctions[slot].appliedAccountStateIndex != 0;
     }
 
+    function getStandingOrderHash(uint16 userId, uint128 nonce) public view returns (bytes32) {
+        return reservedAccountOrders[userId][nonce].orderHash;
+    }
+    
+    function getStandingOrderValidityFrom(uint16 userId, uint128 nonce) public view returns (uint) {
+        return reservedAccountOrders[userId][nonce].validityFrom;
+    }
+
+    function getStandingOrderValidityTo(uint16 userId, uint128 nonce) public view returns (uint) {
+        return reservedAccountOrders[userId][nonce].validityTo;
+    }
+
     /**
      * Auction Functionality
      */
+    function placeStandingSellOrder(
+        uint8[] memory buyToken,
+        uint8[] memory sellToken,
+        uint128[] memory buyAmount,
+        uint128[] memory sellAmount
+    ) public onlyRegistered() {
+        
+        require(auctionIndex < MAX_UINT, "Standing order collection has not yet started");
+
+        // Update Auction Hash based on request
+        uint16 accountId = publicKeyToAccountMap(msg.sender);
```

Comment:
`publicKeyToAccountMap` does not exist anymore. This must be an old instance.

Reply:
`registeredAccounts`

Reply:
it's even used in the placeSellOrder function